mungegithub/github/github.go

```diff
@@ -558,35 +559,39 @@ func (config *Config) NewIssue(title, body string, labels []string, owner string
 
 // Branch returns the branch the PR is for. Return "" if this is not a PR or
 // it does not have the required information.
-func (obj *MungeObject) Branch() string {
-	pr, err := obj.GetPR()
-	if err != nil {
-		return ""
+func (obj *MungeObject) Branch() (string, bool) {
+	pr, ok := obj.GetPR()
+	if !ok {
+		return "", ok
 	}
 	if pr.Base != nil && pr.Base.Ref != nil {
-		return *pr.Base.Ref
+		return *pr.Base.Ref, ok
 	}
-	return ""
+	return "", ok
 }
 
 // IsForBranch return true if the object is a PR for a branch with the given
 // name. It return false if it is not a pr, it isn't against the given branch,
 // or we can't tell
-func (obj *MungeObject) IsForBranch(branch string) bool {
-	objBranch := obj.Branch()
+func (obj *MungeObject) IsForBranch(branch string) (bool, bool) {
```

Comment:
I feel that to return (bool, bool) is confusing, in that one needs to check the result of the second return statement to know if the first is meaningful.